### ISSUE TITLE
Accessibility: All page content should be contained by landmarks for <Multiselect /> Dropdown

### ISSUE THREAD

**Issue by Chinna-Reddy** [ant-design/ant-design#31693]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://4y7bi.csb.app/](https://4y7bi.csb.app/)

### Steps to reproduce
1. Please open the link https://4y7bi.csb.app/ 
2. click on dropdown input field. 
3. once the selection window opens from the dropdown please run the "Accessibility DevTools"(https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) chrome extension from chrome debugger tools to get the accessibility issue - ARIA input fields must have an accessible name

### What is expected?
The dropdown and the content generated in DOM for the Dropdown should be inside the <main> </main> landmark. 

### What is actually happening?
The dropdown and the content generated in DOM  for the Dropdown is NOT inside the <main> </main> landmark.

| Environment | Info |
|---|---|
| antd | 4.16.0 |
| React | 16.12.0 |
| System | Windows 10 |
| Browser | Chrome Version 92.0.4515.131 (Official Build) (64-bit) |

**Comment by zhihongl** [comment]
I have checked this issue by following your instructions, however, I cannot find the issue `ARIA input fields must have an accessible name`.  Are you referring to the issue `All page content should be contained by landmarks`? See the below image for all the warning/errors I get from axe devtools. 

![image](https://user-images.githubusercontent.com/4370435/132259904-61605cfd-b9d0-49ef-9215-d6a9effe3dbe.png)
